script.js

```diff
@@ -0,0 +1,249 @@
+
+  const role = document.createElement('span');
+  role.textContent = projects[i].post;
+  info.appendChild(role);
+
+  const dot2 = document.createElement('img');
+  dot2.src = './folderImg/Counter.svg';
+  dot2.alt = 'dot';
+  info.appendChild(dot2);
+
+  const year = document.createElement('span');
+  year.textContent = projects[i].year;
+  info.appendChild(year);
+
+  const description = document.createElement('p');
+  description.classList.add('card-descrip', 'card-item-mg');
+  description.textContent = projects[i].description;
+  details.appendChild(description);
+
+  const tags = document.createElement('ul');
+  tags.classList.add('card-tags', 'card-item-mg');
+  details.appendChild(tags);
+
+  for (let j = 0; j < projects[i].technologies.length; j += 1) {
+    const tag = document.createElement('li');
+    tag.classList.add('card-tag');
+    tag.textContent = projects[i].technologies[j];
+    tags.appendChild(tag);
+  }
+
+  const action = document.createElement('div');
+  action.classList.add('card-action');
+  details.appendChild(action);
+
+  const button = document.createElement('button');
+  button.classList.add('card-btn');
+  button.id = `card-${projects[i].id}`;
+  button.textContent = 'See Project';
+  action.appendChild(button);
+
+  container.appendChild(card);
+}
+
+const ProjectPopup = (
+  id,
+  title,
+  company,
+  post,
+  year,
+  image,
+  technologies,
+  details,
+  liveVersion,
+  githubLink,
+) => {
+  let technologyList = '';
+  for (let i = 0; i < technologies.length; i += 1) {
+    technologyList += `<li class="card-tag">${technologies[i]}</li>`;
+  }
+
+  return `<div class="pop-head">
+  <div class="pop-head-container">
+    <h3 class="card-title">${title}</h3>
+    <div class="card-info">
+      <span class="info-active">${company}</span>
+      <img src="./folderImg/Counter.svg" alt="dot" />
+      <span>${post}</span>
+      <img src="./folderImg/Counter.svg" alt="dot" />
+      <span>${year}</span>
+    </div>
+  </div>
+  <button id="pop-btn">&times;</button>
+</div>
+<div class="pop-img">
+  <img class="card-img" src="${image}" />
+</div>
+<div class="pop-content">
+  <p class="card-descrip card-item-mg">
+    ${details}
+  </p>
+  <div class="pop-footer">
+    <ul class="card-tags card-item-mg">
+      ${technologyList}
+    </ul>
+    <span class="span"></span>
+    <div class="pop-actions">
+      <button class="pop-button" onclick="window.open('${githubLink}', '_blank')">
+        <span>See live</span>
+        <img src="./folderImg/live-icon.svg" />
+      </button>
+      <button class="pop-button" onclick="window.open('${liveVersion}', '_blank')">
+        <span>See live</span>
+        <img src="./folderImg/github2.svg" />
+      </button>
+    </div>
+  </div>
+</div>`;
+};
+
+projects.forEach((aProject) => {
+  const btnShowProject = document.getElementById(`card-${aProject.id}`);
+  btnShowProject.addEventListener('click', () => {
+    // alert(`clicked on ${aPoject.id}`);
```

Comment:
- [ OPTIONAL ] Please avoid leaving commented pieces of code as it makes your codebase can clutter up the codebase and make it harder to read and understand. This can make it more difficult to identify the important parts of the code and can increase the likelihood of errors or bugs. :100: